designer/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/ProcessTestData.scala

```diff
@@ -134,12 +138,12 @@ object ProcessTestData {
   val validProcessWithEmptySpelExpr: CanonicalProcess =
     validProcessWithParam("fooProcess", "expression" -> Expression.spel(""))
 
-  val validDisplayableProcess: ValidatedDisplayableProcess = toValidatedDisplayable(validProcess)
+  val validDisplayableProcess: DisplayableProcess = toDisplayable(validProcess)
```

Comment:
Just `displayableProcess`?

Reply:
The `valid` prefix in this place and any other in this class is added to show that we prepared valid (correct, without validation errors) scenario. Take a look at `validProcessWithEmptySpelExpr`, `validProcessWithNodeId` etc and `invalidProcess` etc.